.travis.yml

```diff
@@ -55,6 +55,15 @@ cache:
     - $HOME/gcloud/
     - $HOME/site-library
 before_install:
+  #skip push builds if there's an associated PR because we don't look at them anway
+- if [[ ${TRAVIS_EVENT_TYPE} == push ]];
+  then
+    PULL_REQUESTS=$(curl -v -H "Authorization':' token $GITHUB_API_TOKEN" https://api.github.com/repos/broadinstitute/gatk/pulls\?state\=open\&head="broadinstitute:${TRAVIS_BRANCH}");
```

Comment:
Can you create a separate account tied to this token instead of using your personal account?